tools/build_file_generator/BuildFileGenerator.java

```diff
@@ -63,6 +67,7 @@ public BuildFileGenerator(
     this.protoFileAnalyzer = protoFileAnalyzer;
     this.javaClassAnalyzer = javaClassAnalyzer;
     thirdPartyDeps = getThirdPartyDeps();
+    pathsToGenerateBuildFiles = getPathsToGenerateBuildFiles();
```

Comment:
Why do this in the constructor and not in `generateBuildFiles()`?
Actually, also for `thirdPartyDeps = getThirdPartyDeps();`.

Reply:
Done.